eng/pipelines/coreclr/release-tests.yml

```diff
@@ -27,6 +27,18 @@ jobs:
     jobParameters:
       timeoutInMinutes: 120
 
+#
+# Release library builds
```

Comment:
Nit - you should be able to combine CoreCLR and libraries build using the template eng/pipelines/common/build-coreclr-and-libraries.yml, please see e.g.

https://github.com/dotnet/runtime/blob/a773bc3b315f6284f568f20f221b396f7390033c/eng/pipelines/coreclr/jitstress.yml#L21

Reply:
Thanks @trylek. 
I've pushed a [new commit](https://github.com/dotnet/runtime/pull/1596/commits/009ca8dd9db01c72afaf6808d4ae1a8104fb558e) for this change. Please take a look. Thanks.